bot/cogs/snakes.py

```diff
@@ -25,23 +47,183 @@ def __init__(self, bot: AutoShardedBot):
+                snake_info["extract"] = j["query"]["pages"][f"{PAGEID}"]["extract"]
+                snake_info["images"] = j["query"]["pages"][f"{PAGEID}"]["images"]
+                snake_info["fullurl"] = j["query"]["pages"][f"{PAGEID}"]["fullurl"]
+                snake_info["pageid"] = j["query"]["pages"][f"{PAGEID}"]["pageid"]
+            except KeyError:
+                snake_info["error"] = True
+            if snake_info["images"]:
+                i_url = 'https://commons.wikimedia.org/wiki/Special:FilePath/'
+                image_list = []
+                map_list = []
+                thumb_list = []
+                banned = ['Commons-logo.svg',
+                          'Red%20Pencil%20Icon.png',
+                          'distribution',
+                          'The%20Death%20of%20Cleopatra%20arthur.jpg',
+                          'Head%20of%20holotype',
+                          'locator',
+                          'Woma.png',
+                          '-map.',
+                          '.svg',
+                          'ange.',
+                          'Adder%20(PSF).png'
+                          ]
+                for image in snake_info["images"]:
+                    i = image["title"].split(':')[1].replace(" ", "%20")
+                    if not i.startswith('Map'):
+                        for b in banned:
+                            image_banned = False
+                            if b in i:
+                                image_banned = True
+                                break
+                        if image_banned:
+                            log.info("the image is banned")
+                        else:
+                            image_list.append(f"{i_url}{i}")
+                            thumb_list.append(f"{i_url}{i}?width=100")
+                    else:
+                        map_list.append(f"{i_url}{i}")
+            snake_info["image_list"] = image_list
+            snake_info["map_list"] = map_list
+            snake_info["thumb_list"] = thumb_list
+        return snake_info
+
+    @command(name="snakes.get()", aliases=["snakes.get"])
+    @bot_has_permissions(manage_messages=True)
+    @locked()
+    async def get(self, ctx: Context, name: Snake = None):
+        """
+        Fetches information about a snake from Wikipedia.
 
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        if name is None:
+            name = Snake.random()
+
+        data = await self.get_snek(name)
+
+        if data.get('error'):
+            return await ctx.send('Could not fetch data from Wikipedia.')
+
+        match = self.wiki_brief.match(data['extract'])
+        embed = discord.Embed(
+            title=data['title'],
+            description=match.group(1) if match else None,
+            url=data['fullurl'],
+            colour=0x59982F
```

Comment:
What colour is this? Is it not included in the `discord.Colour` class as a staticmethod?

Reply:
No it's just a random green I found online